gtk4/src/expression.rs

```diff
@@ -23,6 +23,14 @@ pub unsafe trait IsExpression:
 }
 
 impl Expression {
+    pub fn upcast(self) -> Self {
```

Comment:
Really not happy with this :(

Reply:
Well, theoretically you should have a trait that implements this on all expression types (including `Expression`), but until then having it as a method on `Expression` (like on all other expression types) would be symmetric at least.

Reply:
We already have those methods on a trait actually, but it is only implemented for the sub-types. I will see what can be done here

Reply:
The problem with going through a trait is auto-generated bindings won't include it :( so we really need to start importing `use glib::prelude::*;` & `use $crate::prelude::*;` instead from gir side.

Reply:
`use $crate::prelude::*` should already include the `glib` prelude or not?

Reply:
it should indeed yes but a task for another time